.gitignore

```diff
@@ -34,6 +34,7 @@ charts/gateway-helm/values.yaml
 
 # dependency charts generated by addons helm.
 charts/gateway-addons-helm/charts/
+charts/gateway-addons-helm/tmpcharts/
```

Comment:
Do we need this?

Reply:
better to keep

Reply:
Curious why this is needed?

Reply:
there's some tmp files, make sure noone will commit it by accident.